README.md

```diff
@@ -125,15 +126,22 @@ $assignmentsStorage = new AssignmentsStorage(
     database: $database,
 );
 /** @var RuleFactoryInterface $rulesContainer */
-$manager = new Manager(
-    itemsStorage: $itemsStorage, 
-    assignmentsStorage: $assignmentsStorage,
-    // Requires https://github.com/yiisoft/rbac-rules-container or other compatible factory.
-    ruleFactory: $rulesContainer,
+$manager = new TransactionalManagerDecorator(
+    new Manager(
+        itemsStorage: $itemsStorage, 
+        assignmentsStorage: $assignmentsStorage,
+        // Requires https://github.com/yiisoft/rbac-rules-container or other compatible factory.
+        ruleFactory: $rulesContainer,
+    ),
 );
 $manager->addPermission(new Permission('posts.create'));
 ```
 
+> Note wrapping manager with decorator - it additionally provides database transactions to guarantee data integrity:
+
+> Note that it's not necessary to use both DB storages. Combining different implementations is possible. A quite popular 
```

Comment:
It should be mentioned here that if used different implementations, then need use `Manager` without decorator.

Reply:
Reverted because when using at least 1 DB storage it's always needed.